src/write.jl

```diff
@@ -47,7 +47,6 @@ end
 
 function edf_write(io::IO, value, byte_limit::Integer)
     edf_value = _edf_repr(value)
-    @assert isascii(edf_value)
```

Comment:
Note that this requirement had to be removed because the version field of a BDF header begins with a non-ASCII byte.